clang/docs/SYCLMainPageforRFC.md

```diff
@@ -0,0 +1,62 @@
+# Add full support for the SYCL programming model
```

Comment:
```suggestion
Discourse topic details:
- Category: [LLVM Project](https://discourse.llvm.org/c/llvm/5)
- Title: "RFC: Add full support for the SYCL programming model"
- Tags: sycl

# Add full support for the SYCL programming model
```

Reply:
The previous RFC was posted to the Clang Frontend Discourse category and all of the other RFCs that we have planned are likewise targeted. I suggest we therefore change the target for this RFC as well.
```suggestion
Discourse topic details:
- Category: [Clang Frontend](https://discourse.llvm.org/c/clang/6)
- Title: "RFC: Add full support for the SYCL programming model"
- Tags: sycl

# Add full support for the SYCL programming model
```